game-buying-guide/.env.example

```
@@ -0,0 +1 @@
+MINO_API_KEY = YOUR_API_KEY
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)